src/RobotBodyFilter.cpp

```diff
@@ -1885,6 +1886,18 @@ bool ScaleAndPadding::operator!=(const ScaleAndPadding& other) const
   return !(*this == other);
 }
 
+template<typename T>
+std::string RobotBodyFilter<T>::getLinkTfPrefix() const
+{
+
+  if (this->linkTfPrefix.empty())
+  {
+    return "";
+  }
+  return this->linkTfPrefix + "/";
```

Comment:
Can you add a check if the last character of linkTfPrefix is already a slash and add it only if it isn't? Or, what is actually the behavior of rviz? Does it add the slashes? Because I can imagine also prefixes like `robot1_`, where appending a slash wouldn't be good.

Reply:
> Does it add the slashes?

Yes, it is a default behavior. (but it is not well-written in anywhere)
when we use `tf_prefix` in rviz RobotModel plugin, it uses `{tf_prefix}/{original_name}` for tf name.

https://github.com/ros-visualization/rviz/blob/b0dae054e27b111758f3b8d3c69ddb4eebaf931f/src/rviz/helpers/tf_prefix.h#L35-L44